gtsfm/utils/geometry_comparisons.py

```diff
@@ -192,3 +194,51 @@ def compare_global_poses(
             for (wTi, wTi_) in zip(wTi_list, wTi_list_)
         ]
     )
+
+
+def compute_relative_rotation_angle(
+    R_1: Optional[Rot3], R_2: Optional[Rot3]
+) -> float:
+    """Compute the angle between two rotations.
+
+    Note: the angle is the norm of the angle-axis representation.
+
+    Args:
+        R_1: the first rotation.
+        R_2: the second rotation.
+
+    Returns:
+        the angle between two rotations.
+    """
+
+    if R_1 is None or R_2 is None:
+        return np.pi
+
+    relative_rot = R_1.between(R_2)
+
+    origin = Rot3()
+
+    relative_rot_axisangle = origin.Logmap(relative_rot)
+    relative_rot_angle = np.linalg.norm(relative_rot_axisangle) % (2 * np.pi)
```

Comment:
I thought rotations in gtsam had an axis-angle method, did you check that?

Reply:
Thanks. Much cleaner!